Library/Homebrew/cmd/link.rb

```diff
@@ -40,16 +42,15 @@ def link
       verbose:   args.verbose?,
     }
 
-    args.named.to_kegs.each do |keg|
+    args.named.to_formulae_and_casks(only: :formula, method: :kegs).freeze.each do |keg|
```

Comment:
What happens here if there's multiple kegs for a formula? How does it pick the right one? They may be sorted, if that helps.